proto/ibc/core/channel/v1/tx.proto

```diff
@@ -261,12 +261,11 @@ message MsgChannelUpgradeInit {
   option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
-  string                    port_id                  = 1;
-  string                    channel_id               = 2;
-  Channel                   proposed_upgrade_channel = 3 [(gogoproto.nullable) = false];
-  ibc.core.client.v1.Height timeout_height           = 4 [(gogoproto.nullable) = false];
-  uint64                    timeout_timestamp        = 5;
-  string                    signer                   = 6;
+  string         port_id         = 1;
+  string         channel_id      = 2;
+  UpgradeTimeout upgrade_timeout = 3 [(gogoproto.nullable) = false];
+  UpgradeFields  upgrade_fields  = 4 [(gogoproto.nullable) = false];
```

Comment:
maybe `timeout` and `fields` are sufficient as names?

Reply:
mega nit here: can we put `upgrade_fields` before `upgrade_timeout`. e.g. swap 3 and 4